Doc/c-api/buffer.rst

```diff
@@ -460,10 +460,12 @@ Buffer-related functions
    :c:func:`PyObject_GetBuffer`.
 
 
-.. c:function:: Py_ssize_t PyBuffer_SizeFromFormat(const char *)
+.. c:function:: Py_ssize_t PyBuffer_SizeFromFormat(const char *format)
 
-   Return the implied :c:data:`~Py_buffer.itemsize` from :c:data:`~Py_buffer.format`.
-   This function is not yet implemented.
+   Return the implied :c:data:`~Py_buffer.itemsize` from :c:data:`~Py_buffer.format`. on success, the
+   size of the return *format*. Otherwise, raise an exception and return -1 on error.
```

Comment:
I don't understand "on success, the size of the return *format*." sentence, and it's redundant with the first sentence, no? I suggest to remove it, and just add:

On error, raise an exception and return -1.